criu/files-reg.c

```diff
@@ -1475,12 +1475,12 @@ static bool store_validation_data_build_id(RegFileEntry *rfe, int lfd)
 	snprintf(buf, sizeof(buf), "/proc/self/fd/%d", lfd);
 	fd = open(buf, O_RDONLY);
 	if (fd < 0) {
-		pr_warn("Build-ID (For validation) could not be obtained for file %s because can't open the file\n",
+		pr_info("Build-ID (For validation) could not be obtained for file %s because can't open the file\n",
```

Comment:
I think this must be an error and we need to stop dumping processes in this case.

Reply:
Why though?
For example, in the ZDTM test "static/remap_dead_pid", the file "/proc/6/mountinfo" could not be opened so only file size was used for validation and the test passed.
If I had put out an error and stopped dumping then the test would've failed.

Log File (From an older commit): https://api.travis-ci.org/v3/job/696469323/log.txt

Reply:
Your example is a special case and I think we don't need to call store_validation_data... in this case.